KeyboardHelperTests/Tests/KeyboardHelperTests.swift

```diff
@@ -9,6 +9,93 @@
 import XCTest
 @testable import KeyboardHelper
 
+class ShowSpyDelegate : KeyboardNotificationDelegate {
```

Comment:
Naming is hard... I called them like that because it's basically the test spy pattern http://xunitpatterns.com/Test%20Spy.html